misk/src/main/kotlin/misk/config/MiskConfig.kt

```diff
@@ -57,7 +68,7 @@ object MiskConfig {
 
     val jsonNode = flattenYamlMap(configYamls)
 
-    val configFile = "$appName-${environment.name.toLowerCase()}.yaml"
+    val configFile = "$appName-${environment.toLowerCase()}.yaml"
```

Comment:
not a new problem, but `toLowerCase()` doesn’t work here or anywhere; you need `toLowerCase(Locale.US)`

https://en.wikipedia.org/wiki/Dotted_and_dotless_I